### ISSUE TITLE
Action.Contract: Add support to human-readable ABI

### ISSUE THREAD

**Issue by marcelomorgado** [tasitlabs/TasitSDK#198]
Refs #82

**Comment by pcowgill** [comment]
@marcelomorgado I don't think we have any urgent need to `Implement a function that returns the human-readable ABI of the contract (even if it was created using json ABI)`.

I think it will be useful to use tools from the ethers.js ecosystem to make sure that any ABIs we track in `tasit-contracts` as ABIs are stored as human-readable ABIs

**Comment by marcelomorgado** [comment]
I see. I've created another issue only for that: https://github.com/tasitlabs/TasitSDK/issues/228 and changed the comment of this one.
